docs/event/metadata.md

```diff
@@ -4,38 +4,79 @@ sidebar_position: 2
 
-#### Extra metadata
-- `extraMetadata: Map<Label, ???>`: additional metadata that Opencast never interprets, but just stores and passes along.<sup>(1?)</sup>
+
+### Extra metadata
+- `extraMetadata: Map<Label, Json>`: additional metadata that Opencast never interprets, but just stores and passes along.
   - The keys of this map consists of a _namespace_ and a _field name_, separated by `:`, i.e. `ns:name`. Both parts must consist of only `a-z`, `A-Z`, `0-9`, `-` and `_`.
-  - The namespace `dct` is special as it refers to the Dublin Core Terms specification, e.g. `dct:rightsHolder` refers to [the `rightsHolder` property](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/rightsHolder) of DC terms. Also see [the DC mapping section below](#dublin-core-mapping). It should be avoided to set fields that already have a mapping, like `dct:title`, which is mapped to the OC core metadata `title`.
+  - The values of this map are arbitrary JSON values, i.e. Booleans, numbers, strings, arrays, maps or `null`. Items of arrays & maps can recursively be any JSON value as well.
   - Unlike the "extended metadata" before, using `extraMetadata` does work out of the box and does not incur any relevant performance overhead. Therefore, applications are encouraged to add useful data here, e.g. `studip:course-id`, `oc-studio:version` or `ethz:room-number`.
-  - There should be a community resource for collecting used fields and best practices around `extraMetadata`. That way, common requirements are identified quickly and the community can converge towards a standard.
+
+#### Community documentation
+
+There should be a community document for specifying rules, as well as collecting used fields and best practices around `extraMetadata`.
+That way, common requirements are identified quickly and the community can converge towards a standard.
+
+> Official namespaces:
+> - **`oc`**: reserved.
+> - **`ocx`**: OC eXperimental for fields that might get promoted to core metadata field in the future.
+>   - `ocx:downloadable: bool`: a flag indicating whether users are allowed to download this video (i.e. tracks attached to this event). This can inform external apps whether to show a download button or to enable anti-download protection. The exact effects of this flag are deliberately unspecified, this merely states an *intent*.
+>   - `ocx:listed: bool`: specifies whether this event should be considered "list", meaning that users can find it via search. If it is `false`, users have to know the ID of the event (e.g. via a series or playlist) in order to access it.
+>   - `ocx:explicitContent: bool`: specifies whether this event contains content that is considered "explicit", like swear words or whatnot. This is required for some integrations like iTunes.
```

Comment:
I'm still curious where we integrate with iTunes 🙈

Reply:
Didn't I answer that before? Tobira offers RSS feeds for series, and some universities want to put those RSS feeds in itunes somehow. And iTunes requires the `itunes:explicit` flag to be set. Also see https://github.com/elan-ev/tobira/issues/1396

Reply:
Sorry. Now that I read your comment, I remember you saying this.